src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -25,48 +25,89 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
-### Verify artifact names match sourceMappingURL
+### Verify artifact names match `sourceMappingURL` value
 
-When [uploading source maps to Sentry]({%- link _documentation/platforms/javascript/sourcemaps/availability.md -%}), you must name your source map files with the same name found in `sourceMappingURL`.
+The `sourceMappingURL` comment on the last line of your bundled or minified JavaScript file tells Sentry (or the browser) where to locate the corresponding source map. This can either be a fully qualified URL, a relative path, or the filename itself. When uploading artifacts to Sentry, you must name your source map files with the same name found in the  `sourceMappingURL` comment.
 
-For example, if you have the following in a minified application file, `app.min.js`:
+i. e. if your file is similar to:
 
 ```javascript
-//-- end app.min.js
-//# sourceMappingURL=https://example.com/dist/js/app.min.js.map
+    // -- end script.min.js
+    //# sourceMappingURL=script.min.js.map
 ```
 
-Sentry will look for a matching artifact named exactly `https://example.com/dist/js/app.min.js.map`.
+then your uploaded artifact should also be named `script.min.js.map`. 
 
-Note also that Sentry will resolve relative paths. For example, if you have the following:
+or if your file is similar to: 
 
 ```javascript
-// -- end app.min.js (located at https://example.com/dist/js/app.min.js)
-//# sourceMappingURL=app.min.js.map
+    //-- end script.min.js
+    //# sourceMappingURL=https://example.com/dist/js/script.min.js.map
 ```
 
-Sentry will resolve `sourceMappingURL` relative to `https://example.com/dist/js/` (the root path from which `app.min.js` was served). You will again need to name your source map with the full URL: `https://example.com/dist/js/app.min.js.map`.
-
-If you serve the same assets from multiple origins, you can also alternatively use our tilde (~) path prefix to ignore matching against protocol + hostname. In which case, `~/dist/js/app.min.js.map`, will also work. See: [Assets Accessible at Multiple Origins]({%- link _documentation/platforms/javascript/sourcemaps/availability.md -%}#uploading-source-maps-to-sentry).
+then your uploaded artifact should also be named `https://example.com/dist/js/script.min.js.map`
 
 
 ### Verify artifact names and file path are correct
```

Comment:
What does "correct" mean in this context? Matching? Also, are the file paths in question the ones in the stacktrace?